### ISSUE TITLE
3D DataSet Visualization Dialog behaviour

### ISSUE THREAD

**Issue by anhi** [BALL-Project/ball#165]
**Reported by akdehof on 28 Feb 39038456 07:06 UTC**
All grids created by "Render Grid" in the Mouse Menu of the Dataset-Window are assigned the same name "Grid Volume" in the Representations-Window, which leads to a lot confusion. This effect is  amplified since the order of the representations in the Representations-Window seem to randomly change. 
Maybe specifying the new name directly in the dialog helps.

Additionally, if one dataset is highlighted (or selected ??? the one which equals making the entry blueish) and one opens the Mouse Menu and selects "Render Grid", then in the following dialog "Grid Visualization" the preselected grid is __not__ necessarily the one, that was previously "blueished". Not noting this leads again to a lot confusion :-)

**Comment by anhi** [comment]
**Commented by dstoeckel on 22 Sep 39054867 16:26 UTC**
Release 1.2.1 was cancelled, reassigning to 1.3

**Comment by anhi** [comment]
**Commented by Marcel on 25 Oct 39837811 16:53 UTC**
Yeah, that was _really_ annoying! ;-)
Anyway, it is fixed now.

(By the way, this was really a defect and not some missing functionalty...)

**Comment by anhi** [comment]
**Commented by Marcel on 13 Jun 39854948 11:33 UTC**
I now noticed a similar problem for visualization of contour surfaces.
This dialog also lets you choose the grid by means of a combobox at the top.
But, on the one hand the dataset that is selected when opening the dialog is wrong (in case of more than one loaded grid) and one the other hand choosing a different grid from this combobox did not change anything at all! :-D

Anyway, I fixed this now as well.

But perhaps we should remove those stupid comboboxes from the both dialogs (grid and contour visualization).
All they do is introduce confusion! The user chooses the grid he wants to visualize by selecting it from the data controller anyway ...
